src/utils.ts

```diff
@@ -266,3 +266,19 @@ export function searchLinkByRel(links: Link[], rel: string): Link[] {
   var regularExpression = new RegExp('\\b' + escapeRegExp(rel) + '\\b');
   return links.filter((l) => regularExpression.test(l.rel));
 }
+
+export function intlFormatNumber(lang: string, format: string, value: number) {
+  switch (format) {
+    case 'compact':
+      return new Intl.NumberFormat('en', {
```

Comment:
For `compact` format you always use locale `en`, so the only option that really uses language option is the intl option, I think here perhaps it's best to have two separate functions. 
`compact` --> compactNumberFormat
`intl` --> languageNumberFormat (this one depends only on language)

Reply:
Yes, it could be separated.
I was thinking that there's no too many differences between use this function and use directly the Intl object. But I thought that maybe, in a future, we could want to add a new behavior (as return short format only for numbers greater than 9999, for example) and in this case it would be annoying to apply it in every place that we use it. 

I was thinking too, that maybe, in a future, we could also want to show a short format based on the language (for Spanish, for example, `1000` would be `1 mil`). Beyond that, it could be in separated functions anyway.